test/e2e/webhook_test.go

```diff
@@ -107,4 +108,40 @@ var _ = Describe("webhook tests for CRP UPDATE operations", Ordered, func() {
 			return nil
 		}, testutils.PollTimeout, testutils.PollInterval).Should(Succeed())
 	})
+
+	It("should deny update on CRP with invalid placement policy for PickAll", func() {
```

Comment:
Strictly speaking we don't really need this e2e. Doesn't hurt to have one but why not pick something specific error related to the pickAll?

Reply:
Agreed, we don't actually need the E2Es but I wanted to at least cover some important cases in E2Es that are already covered by UTs just to provide solid proof

I am testing a bad label selector within cluster affinity which is within RequiredDuringSchedulingIgnoredDuringExecution which is specific to PickAll, I also added topology constraints which is not allowed for PickAll